index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
+  </body>
+</html>
```

Comment:
### Your Readme File
![readme](https://user-images.githubusercontent.com/93039359/185210161-6be3008b-3d5f-4bfb-9208-8c31b388e6f7.png)

- [ ] You didn't have a descriptive readme file for you project. :blush: You can add it from this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) :+1: